libs/pxweb2-ui/src/lib/components/Typography/Label/Label.tsx

```diff
@@ -0,0 +1,36 @@
+import cl from 'clsx';
+import classes from './Label.module.scss';
+
+export interface LabelProps extends React.HTMLAttributes<HTMLParagraphElement> {
+  size?: 'medium' | 'small';
+  align?: 'start' | 'center' | 'end';
+  textcolor?: 'default' | 'subtle';
+  visuallyHidden?: boolean;
+  children?: React.ReactNode;
+}
+
+export function Label({
+  size = 'medium', 
+  align = 'start',
+  textcolor = 'default',
+  visuallyHidden = false,
+  children,
+  ...rest
+}: LabelProps) {
+  return (
+    <p 
```

Comment:
This should probably be a span. A `<p>` tag is for paragraphs

Reply:
@michaelpande 

Now is a label element with the possibility to set htmlFor:

>       <Label htmlFor="fname">First name:</Label><br />
>       <input type="text" id="fname" name="fname" defaultValue="John" /><br />
>